src/Stratis.Bitcoin.FullNode.sln

```diff
@@ -158,15 +158,17 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.Features.FederatedP
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.Features.FederatedPeg.Tests", "Stratis.Features.FederatedPeg.Tests\Stratis.Features.FederatedPeg.Tests.csproj", "{DE629FE5-BC8A-4597-B19C-DDAB5EC1528D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.CirrusD", "Stratis.CirrusD\Stratis.CirrusD.csproj", "{D1A3A36B-4F50-4C10-9DAE-DACFAF617A6D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.CirrusD", "Stratis.CirrusD\Stratis.CirrusD.csproj", "{D1A3A36B-4F50-4C10-9DAE-DACFAF617A6D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.CirrusPegD", "Stratis.CirrusPegD\Stratis.CirrusPegD.csproj", "{3E6D4D9E-CAF5-4A23-9C96-07B1B1FE76B7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.CirrusPegD", "Stratis.CirrusPegD\Stratis.CirrusPegD.csproj", "{3E6D4D9E-CAF5-4A23-9C96-07B1B1FE76B7}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.Features.FederatedPeg.IntegrationTests", "Stratis.Features.FederatedPeg.IntegrationTests\Stratis.Features.FederatedPeg.IntegrationTests.csproj", "{D71A7BBE-E64A-4F2E-B2A9-6D742FE94ED9}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Benchmarks", "Benchmarks", "{7FFED910-450B-4D6E-9085-6E5A6731D72A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stratis.Benchmark", "Stratis.Benchmark\Stratis.Benchmark.csproj", "{B8BC6552-6409-4BE1-B085-E8E6513EC251}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.Benchmark", "Stratis.Benchmark\Stratis.Benchmark.csproj", "{B8BC6552-6409-4BE1-B085-E8E6513EC251}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stratis.FederatedSidechains.AdminDashboard", "Stratis.FederatedSidechains.AdminDashboard\Stratis.FederatedSidechains.AdminDashboard.csproj", "{516F4F2E-587C-4C55-91A3-C552A34C0CC8}"
```

Comment:
I have added the Sidechain Dashboard project in the `Apps` folder

![image](https://user-images.githubusercontent.com/5221349/56475357-5a4eff80-6487-11e9-9da0-e45e7006378d.png)